configs/eln_extras_automotive.yaml

```diff
@@ -0,0 +1,15 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Automotive packages
+  description: ELN Extras packages that the CentOS Automotive SIG cares about
+  maintainer: sig-automotive
+  packages:
+    - qm
+  arch_packages:
+    x86_64:
+      - vhost-device-*
```

Comment:
vhost-device-* packages are built in koji for rust-vhost-device* src.rpm. Is it correct here listing the built binaries rpms? Or should I add here the package name as defined in koji/src.rpm?

Reply:
RPM name is correct for `packages` and `arch_packages`.